From b7e1b1ff12fa93f541f843ed9c76bb52fec997a4 Mon Sep 17 00:00:00 2001
From: Geoffrey Papaux <geoffrey.papaux@gmail.com>
Date: Fri, 21 Mar 2014 16:30:59 +0100
Subject: [PATCH] Corrects the use of a deprecated typedef

---
 tools/lvm.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/lvm.c b/tools/lvm.c
index e002a76..5fa35d2 100644
--- a/tools/lvm.c
+++ b/tools/lvm.c
@@ -186,7 +186,7 @@ int lvm_shell(struct cmd_context *cmd, struct cmdline_context *cmdline)
 	char *input = NULL, *args[MAX_ARGS], **argv;
 
 	rl_readline_name = "lvm";
-	rl_attempted_completion_function = (CPPFunction *) _completion;
+	rl_attempted_completion_function = (rl_completion_func_t *) _completion;
 
 	_read_history(cmd);
 
-- 
1.7.9.5

